packages/engine/tests/examples/mod.rs

```diff
@@ -0,0 +1,131 @@
+#![cfg(not(debug_assertions))]
+
+use crate::run_test;
+
+// https://core.hash.ai/@hash/city-infection-model/6.4.2
+mod city_infection_model {
+    use crate::run_test;
+
+    // Rust behavior is currently not supported
+    run_test!(city_infection_model, experiment: infected_linspace, #[ignore]);
+    // Rust behavior is currently not supported
+    run_test!(city_infection_model, experiment: duration_range_monte_carlo, #[ignore]);
+}
+
+// https://core.hash.ai/@hash/sugarscape/7.5.0
+mod sugarscape {
+    use crate::run_test;
+
+    // optimization experiment is not implemented
+    run_test!(sugarscape, experiment: minimize_the_gini_coefficient, #[ignore]);
```

Comment:
Lets just not include optimization run_test! calls. 

(Still leave the #[ignore] on the Rust behaviors for now though)